### PR TITLE
TFM Issue Triage Documentation

### PR DESCRIPTION
**Pull request by mbilalamjad**
# Overview/Summary

Documentation for TFM Issue Triage 

## This PR fixes/adds/changes/removes

NA

### Breaking Changes

NA

## As part of this Pull Request I have

- [x] Read the Contribution Guide and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [x] Ensured PR tests are passing
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+## Module Owner Responsibilities
+
+During the triage process, module owners are expected to check, complete and follow up on the items described in the sections below.
+
+Module owners **MUST** meet the SLAs defined on the [Module Support](/Azure-Verified-Modules/help-support/module-support/) page! While there's automation in place to support meeting these SLAs, module owners **MUST** check for new issues on a regular basis.
+<!--
+{{< hint type=important >}}
+The [BRM repository](https://aka.ms/BRM) includes other, non-AVM modules and related GitHub issues. As a module owner, make sure you're only triaging, managing or otherwise working on issues that are related to AVM modules!
+{{< /hint >}}
+-->
```
Comment (segraef) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:41]: comment could be removed

**Review by segraef — DISMISSED** [review]
lgtm

**Review by PmeshramPM — DISMISSED** [review]
checked and all good

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+This page provides guidance for **Terraform Module (TFM) owners** on how to triage **AVM module issues** and **AVM question/feedback** items filed in their Terraform Module Repo, as well as how to manage these GitHub issues throughout their lifecycle.
+
+As such, **the following issues are to be filed in the TFM repository**:
+
+- **AVM Module Issue**: Issues specifically related to an existing AVM module, such as feature requests, bug and security bug reports.
+- **AVM Question/Feedback**:Generic feedback and questions, related to existing AVM module, the overall framework, or its automation (CI environment).
+
+Do **NOT** file the following types of issues in the **TFM repository**, as they **MUST** be tracked in the **AVM repo**:
+
+- **\[[Module Proposal](https://aka.ms/AVM/ModuleProposal)]**: Proposals for new AVM modules, or for migrating existing CARML/TFVM modules to AVM.
```
Comment (matebarabas) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:22]:
```suggestion
- **\[[Module Proposal](https://aka.ms/AVM/ModuleProposal)]**: Proposals for new AVM modules.
```

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+geekdocAlign: left
+geekdocAnchor: true
+geekdocToC: 1
+---
+
+{{< toc >}}
+
+## Overview
+
+This page provides guidance for **Terraform Module (TFM) owners** on how to triage **AVM module issues** and **AVM question/feedback** items filed in their Terraform Module Repo, as well as how to manage these GitHub issues throughout their lifecycle.
```
Comment (matebarabas) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:13]:
```suggestion
This page provides guidance for **Terraform Module owners** on how to triage **AVM module issues** and **AVM question/feedback** items filed in their Terraform Module Repo(s), as well as how to manage these GitHub issues throughout their lifecycle.
```

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+---
+title: TFM Issue Triage
```
Comment (matebarabas) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:2]:
```suggestion
title: Terraform Issue Triage
```

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+## Overview
+
+This page provides guidance for **Terraform Module (TFM) owners** on how to triage **AVM module issues** and **AVM question/feedback** items filed in their Terraform Module Repo, as well as how to manage these GitHub issues throughout their lifecycle.
+
+As such, **the following issues are to be filed in the TFM repository**:
+
+- **AVM Module Issue**: Issues specifically related to an existing AVM module, such as feature requests, bug and security bug reports.
+- **AVM Question/Feedback**:Generic feedback and questions, related to existing AVM module, the overall framework, or its automation (CI environment).
+
+Do **NOT** file the following types of issues in the **TFM repository**, as they **MUST** be tracked in the **AVM repo**:
```
Comment (matebarabas) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:20]:
```suggestion
Do **NOT** file the following types of issues in a **Terraform repository**, as they **MUST** be tracked in the **AVM repo**:
```

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+{{< /hint >}}
+
+<br>
+
+## Module Issue
+
+An issue is considered to be an "AVM module issue" if
+
+- it was opened through the **AVM Module Issue** template in the TFM repository,
```
Comment (matebarabas) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:55]:
```suggestion
- it was opened through the **AVM Module Issue** template in the Terraform repository,
```

File: docs/content/help-support/issue-triage/tfm-issue-triage.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+5. When more detailed plans are available, communicate expected timeline for the update/fix to the requestor (issue author).
+6. Only close the issue, once the next version of the module was fully developed, tested and published.
+
+<br>
+
+## General Question/Feedback and other standard issues
+
+An issue is considered to be an "AVM Question/Feedback" if
+
+- it was opened through the **AVM Question/Feedback** template in your TFM repository,
```
Comment (matebarabas) [inline docs/content/help-support/issue-triage/tfm-issue-triage.md:88]:
```suggestion
- it was opened through the **AVM Question/Feedback** template in your Terraform repository,
```

**Review by matebarabas — APPROVED** [review]
Reviewed and updated together with Bilal, ready to be released.

**Pull request merged**